### PR TITLE
created header and headline

### PR DESCRIPTION
**Pull request by summyalena**
Hello Microverse.
In this feature, I added and created the header and headline of the website portfolio from the one of the templates gotten from figma. 
I also added linters too to check in for errors

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="index.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;400&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap" rel="stylesheet"/>
+    <title>Document</title>
+</head>
+<body>
+    <div class="body">
+        <header id="header">
```
Comment (Meltrust) [inline index.html:15]: - [ ] Please, add more bottom margin to the header section, as on the Figma design.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+            <div class="wrapper">
+                <div class="logo">
+                    <a>Alena Okolike</a>
```
Comment (Meltrust) [inline index.html:18]: - [ ] This section should have the font family "Roboto", as on the Figma design, but it's displaying "Times new roman".  Kindly correct this.
- [ ] This section line-height should be 24 px.  Kindly correct this.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+                        <path fill-rule="evenodd" clip-rule="evenodd" d="M0 1C0 0.447715 0.447715 0 1 0H17C17.5523 0 18 0.447715 18 1C18 1.55228 17.5523 2 17 2H1C0.447715 2 0 1.55228 0 1ZM0 6C0 5.44772 0.447715 5 1 5L17 5C17.5523 5 18 5.44772 18 6C18 6.55229 17.5523 7 17 7L1 7C0.447715 7 0 6.55228 0 6ZM1 10C0.447715 10 0 10.4477 0 11C0 11.5523 0.447715 12 1 12H17C17.5523 12 18 11.5523 18 11C18 10.4477 17.5523 10 17 10H1Z" fill="white"/>
+                        </svg>
+                </nav>
+            </div>
+        </header>
+
+        <section class="main">
+            <div class="mainwrap">
+                <div class="heading">
+                    <h1>Hey There. I'am Assumpta</h1>
```
Comment (Meltrust) [inline index.html:31]: - [ ] This text should have the font family "Crete round", kindly correct this.  
- [ ] This text's font-size and line-height does not match the Figma.  Kindly correct this.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+                </div>
+                <div class="mid">
+                    <h4>I'm a Software Developer</h4>
```
Comment (Meltrust) [inline index.html:34]: - [ ] This text font family should be "Roboto", but it is rendering "Times new roman". Kindly correct this.
- [ ] font-size should be 27px, line-height 32px, according to the Figma.  Kindly correct this.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,92 @@
...
+                </div>
+                <div class="para">
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,
```
Comment (Meltrust) [inline index.html:37]: - [ ] This text element should have the font-family "Poppins" but it is rendering "Times new roman".  Kindly correct this.
- [ ] This text element should have the line-height 24px, according to the Figma.  Kindly correct this.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="index.css"/>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Inter:wght@200;400&display=swap" rel="stylesheet">
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@400&display=swap" rel="stylesheet"/>
+    <title>Document</title>
+</head>
+<body>
```
Comment (Meltrust) [inline index.html:13]: - [ ] The margins do not match the Figma template.  Look: 

Your Implementation | Figma Design
:---:|:---:
image | image

Kindly correct your margins and move the background image so it goes lower, as in the Figma design.

That way, your project will look like the Figma 💪

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @summyalena

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight:

- Nice start, colors match✔️ 

You can do it

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (summyalena) [reply index.html:13]: Hello Mr Meltrust. Thank you for your review. The actual heading font size from figma is too large on my personal project. Please can I adjust it ?

**Pull request merged**